src/backend/common/suggestions/media_parser.py

```diff
@@ -25,7 +25,7 @@ class MediaParser:
         MediaType.INSTAGRAM_IMAGE,
     }
 
-    OEMBED_PROVIDERS: Set[MediaType] = {MediaType.INSTAGRAM_IMAGE}
+    OEMBED_PROVIDERS: Set[MediaType] = set()
```

Comment:
Since the image links expire, it doesn't make sense to call the API here anymore.